docs/lib/datastore/fragments/js/datastore-events.md

```diff
@@ -1,5 +1,3 @@
-For instance, to listen to see if the network status is active, you could set up the following listener:
```

Comment:
What is the need for removing this? Are we placing a fragment here?

Reply:
It was a change according to Daniel's suggestion. see [this comment](https://github.com/aws-amplify/docs/pull/2470#discussion_r497881887)

Reply:
@drochetti Can you weigh in on this? I don't want to remove the line from JS if there is no replacement for it. I am seeing that the iOS and Android use cases have the extend paragraph but if we aren't going to add it for JS I dont want this removed.

Reply:
My suggestion in that comment was to add an introductory paragraph and then share the content across _all_ platforms.

I see the changes are only shared in a `native_common` fragment. Any reason not to allow JS to use the same content @ruiguoamz?

Reply:
I don't think there is reason not to allow JS to use the same content. The description should be the same for 3 platforms, only the code snippets are different